### DISCUSSION TITLE
LTeX+ Not Correcting Grammar or Spelling Mistakes Beyond Basic Punctuation

### DISCUSSION THREAD

**Discussion by Pelayoprogrammer** [ltex-plus/vscode-ltex-plus#95] [Q&A]
Hello,
I am experiencing an issue with LTeX+ in Visual Studio Code. It seems the extension only corrects basic issues like capitalization after periods and punctuation mistakes, but it does not identify or correct grammar mistakes or misspelled words. I have not opened an issue because I do not know if it is a bug or simply the result of my ignorance.

**Comment by spitzerd** [comment]
Are you using a specific variant like en-US or de-DE instead of the generic language code like en or de?

Does [LanguageTool.org](https://languagetool.org) displays spelling mistakes for your text?

**Reply by Pelayoprogrammer** [reply]
Thanks for your answer!
I'm using the standard configuration, and specifically en-US  as language. [LanguageTool.org](https://languagetool.org/) displays spelling mistakes for my text, as I'm testing with basic misspelled words.

**Comment by spitzerd** [comment]
Ok, then you have to open an issue and fill in the issue template so I can investigate the issue.
